velox/dwio/common/Options.h

```diff
@@ -622,6 +622,8 @@ struct WriterOptions {
   tsan_atomic<bool>* nonReclaimableSection{nullptr};
   std::optional<velox::common::CompressionKind> compressionKind;
   std::optional<uint64_t> orcMinCompressionSize{std::nullopt};
+  std::function<std::shared_ptr<dwio::common::FlushPolicy>()>
```

Comment:
Why this needs to a shared_ptr instead of a unique one? Thanks!

Reply:
@xiaoxmeng Previously, there was an issue when converting std::unique_ptr<dwio::common::FlushPolicy> to std::unique_ptr<dwrf::FlushPolicy>, which led to the use of std::shared_ptr instead. However, it seems that this issue has been resolved now. I have refined the code, and it has been successfully converted to std::unique_ptr.